src/snowflake/cli/api/project/schemas/entities/common.py

```diff
@@ -59,6 +61,27 @@ def get_type(cls) -> str:
         return cls.model_fields["type"].annotation.__args__[0]
 
     meta: Optional[MetaField] = Field(title="Meta fields", default=None)
+    identifier: Optional[Union[Identifier | str]] = Field(
```

Comment:
shouldn't this be required?

Reply:
Per design if identifier is not specified we fall-back to entitiy id